controllers/user.controller.js

```diff
@@ -0,0 +1,48 @@
+const User = require("../models/user.model");
+
+exports.create = function(req, res){
+    let user = new User(
+        {
+            username: req.body.username,
+            password: req.body.password
+        }
+    );
+
+    user.save(function (err) {
+        if (err)
+        {
+            res.send("Error");
+        }
+        else
+        {
+            res.send('User created successfully');
```

Comment:
use ```res.status()``` to give the HTTP status code as well (this should be applied to all routes)